org.eclipse.jdt.core/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: %pluginName
 Bundle-SymbolicName: org.eclipse.jdt.core; singleton:=true
-Bundle-Version: 3.33.0.qualifier
+Bundle-Version: 3.33.100.qualifier
```

Comment:
I'm pretty sure this is not needed because the version was already updated for 4.27

Reply:
Wont we need a bump for JDT debug to require and use the newer version? Since we plan to change code there to use this new method.

Reply:
no